server/app/views/admin/questions/QuestionTranslationView.java

```diff
@@ -170,11 +176,14 @@ private Optional<DomContent> multiOptionQuestionFields(
       QuestionOption option = options.get(optionIdx);
       optionFieldsBuilder.add(
           fieldWithDefaultLocaleTextHint(
-              FieldWithLabel.input()
+              FieldWithLabel.textArea()
```

Comment:
suggestion (maybe blocking): I think we should actually keep the options field as an input and say "Some markdown is supported" since it's an input field in the question editor: https://github.com/civiform/civiform/blob/2af469ff7d1408f1cb08b0da36b5c08c9d16ca3d/server/app/views/admin/questions/QuestionConfig.java#L258

<img width="546" alt="image" src="https://github.com/user-attachments/assets/30d036bb-fd12-4e8b-baaa-c9f05fc8b42d">

Reply:
I have updated the code and removed markdown on from it.